### PR TITLE
NatGen-base.commons-lang.mid-2127.idx-16820.1.mutant

### PR DESCRIPTION
**Pull request by ahmedfahad04**
fix indentation max() - IEEE754rUtils.java

### CODE REVIEW

File: commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java [hunk 1]
```diff
@@ -166,7 +168,7 @@ public static double max(final double... array) {
      * @param array  an array, must not be null or empty
      * @return the minimum value in the array
      * @throws NullPointerException if {@code array} is {@code null}
-     * @throws IllegalArgumentException if {@code array} is empty
+     * 5698.3.mutant codebert-base.commons-collections.mid-2586.idx-5698.3.buggy @throws IllegalArgumentException if {@code array} is empty
```
Comment (faustinoaq) [inline commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java]: Is this change required? needs more review

File: commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java [hunk 2]
```diff
@@ -151,6 +151,8 @@ public static float min(final float a, final float b) {
     public static double max(final double... array) {
         Objects.requireNonNull(array, "array");
         Validate.isTrue(array.length != 0, "Array cannot be empty.");
+        
+        // find and return max
```
Comment (faustinoaq) [inline commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java]: Is this comment required?

**Review by faustinoaq — CHANGES_REQUESTED** [review]

File: commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java (hunk 2 above)
Comment (darko-marinov) [reply commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java]: No, please make no other changes but just reformat the code.

File: commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java (hunk 1 above)
Comment (ahmedfahad04) [reply commons-lang/src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java]: According to the evaluation task, we were told to work on only the part of the code which is not `formatted` at all. And the change in **171th** line is not within that scope, so I ignored it. I think we can ignore that comment for that PR.

**Comment by darko-marinov** [comment]
Please revise your PR to minimize the changes from `main` while preserving the mutant.

**Comment by ahmedfahad04** [comment]
> Please revise your PR to minimize the changes from `main` while preserving the mutant.

I have revised my PR. Now it aligns with the `main` branch preserving the mutant.

**Comment by faustinoaq** [comment]
> > Please revise your PR to minimize the changes from `main` while preserving the mutant.
> 
> I have revised my PR. Now it aligns with the `main` branch preserving the mutant.

Does this one needs formatting at all? All I see is this adding 1 comment line @ahmedfahad04 

image

**Comment by darko-marinov** [comment]
There're unnecessary white-space changes on lines 154 and 160: https://github.com/ahmedfahad04/UIUCPlus/compare/461dfea3626452643d4138fef934535979fe87e9..NatGen-base.commons-lang.mid-2127.idx-16820.1.mutant

**Comment by ahmedfahad04** [comment]
> > > Please revise your PR to minimize the changes from `main` while preserving the mutant.
> > 
> > I have revised my PR. Now it aligns with the `main` branch preserving the mutant.
> 
> Does this one needs formatting at all? All I see is this adding 1 comment line @ahmedfahad04 
> 
> image
> 

I just wanted to keep the code aligned with the `main`. Hence I kept the comment as its present in the `main` branch.

**Comment by ahmedfahad04** [comment]
> There're unnecessary white-space changes on lines 154 and 160: https://github.com/ahmedfahad04/UIUCPlus/compare/461dfea3626452643d4138fef934535979fe87e9..NatGen-base.commons-lang.mid-2127.idx-16820.1.mutant

Fixed ✅
